RockWeb/Themes/NewSpring/Scripts/smooth-scroll.js

```diff
@@ -1,10 +1,81 @@
 /*!
- * smooth-scroll v14.2.1: Animate scrolling to anchor links
- * (c) 2018 Chris Ferdinandi
+ * smooth-scroll v16.0.3
```

Comment:
Looks like you just copy and pasted a new version of this script on top of the old one, is that right?

Reply:
Correct, updated to the lastest version.